### ISSUE TITLE
Too great

### ISSUE THREAD

**Issue by howtomakeaturn** [happypoulp/redux-tutorial#75]
The tutorial is too great.

It makes me skip my dinner and sleep, keep reading it.

This is not good for my health. How should I deal with this issue?

Thanks a lot for the awesome tutorial. 

You really know how to teach & write.

**Comment by happypoulp** [comment]
:laughing: !! You made my day, thanks a lot! May your learning motivation last long, and sorry for your health...

**Comment by zereraz** [comment]
Yeah great tutorial! I didn't sleep all night, read it! :smile: 
This had a good mix of internals of a library code with the explanation and concepts!

**Comment by lukephills** [comment]
This tutorial was so brilliant I dreamt about it all night. Thank you!

**Comment by pmkwilliams** [comment]
I'm just going to pile on here. 
Thank you! Great walkthrough. I finally feel like I understand react+redux concepts.
I will be sending you a list of everything I want to learn from now on so you can write guides for me. :stuck_out_tongue:
But seriously keep up the good work.

**Comment by happypoulp** [comment]
:heart:

**Comment by Mickael-van-der-Beek** [comment]
This tutorial saved my marriage and I'm not even married.

**Comment by SOSANA** [comment]
I have read blogs, docs, tutorials and by far this was the best way to actually learn and understand redux. Had so many "ah ha" moments and just want to say thank you @happypoulp for sharing and explaining redux. Wish I found this sooner. Amazing tutorial, highly recommended.

**Comment by cjke** [comment]
Thank you. This tutorial actually made everything finally click with Redux.

**Comment by xsstomy** [comment]
Thank you very much , it makes  me  understand the flux concepts and MVC ,i read a lot articles about flux and MVC ,but i didn't  understand that . really thank you very much

**Comment by timklimowicz** [comment]
Agreed, thanks so much for this. I've spent the last few days reading as much as possible about Flux & Redux, and everything I came across made it seem so convoluted and very hard to grasp. This really showed things to be just as simple as everyone was saying (but not showing well enough), and it all _finally_ fully clicked. Thank you.

**Comment by ToraSonic** [comment]
One of the most concise and clear tutorials on redux I have read. Thanks for doing this!
